core/src/main/res/values/strings.xml

```diff
@@ -321,4 +321,5 @@
   <string name="open_article">Open Article</string>
   <string name="delete_note_dialog_message">Note: Notes are not deleted from your storage</string>
   <string name="delete_selected_notes">Delete Selected Notes?</string>
+  <string name="missing_split_version_for_objectbox">Un-Supported architecture unable to open please install this app from PlayStore</string>
```

Comment:
>if user are using the device architecture which is not supported then i think there is no way he can use the app so there is not point for navigating to playstore

Maybe a meaningful message? If the device architecture isn't supported then what is the point of installing the app from the play store?